src/media/js/views/langpacks.js

```diff
@@ -0,0 +1,40 @@
+define('views/langpacks',
+    ['l10n', 'log', 'models', 'urls', 'z'],
+    function(l10n, log, models, urls, z) {
+    'use strict';
+
+    var gettext = l10n.gettext;
+    var logger = log('langpacks');
+
+    function transform_to_app(langpack) {
```

Comment:
Technically, langpacks are really apps though - privileged, packaged apps. they aren't using the `Webapp` model in zamboni (and therefore aren't exposed as apps) because what we do with them is pretty specific (no reviews, no ratings, don't want them to appear elsewhere, etc)